docs/changelog.rst

```diff
@@ -1,8 +1,37 @@
 =========
 Changelog
 =========
-
 * :release:`0.100.3-rc5 <2019-05-08>`
+* :bug:`4080` Fix "Unexpected signature format" issue in matrix transport
+* :feature:`4043` Update raiden-contracts to 0.19.0 with Görli Testnet support
+* :bug:`4041` Fix V20 -> V21 migration chain-state transformation to canonical identifiers
+* :bug:`4024` Fix clearing a channel state if and only if all unlocks are done by channel participants
+* :feature:`4002` Nightly builds are now deployed to Docker Hub
```

Comment:
This is an infrastructure change, not related to the raiden client software itself.